static-site/components/list-resources/group-and-resource-links.module.css

```
@@ -0,0 +1,13 @@
+.baseLink {
+   color: #5097BA;
+   text-decoration: none;
+}
+.baseLink:hover {
+  color: #31586c;
+}
```

_nit_

Noticed in preview app that the links have underlines on hover. 
Seems like an explicit `text-decoration: inherit` is needed override the [global.css](https://github.com/nextstrain/nextstrain.org/blob/90f10e914242103d90bb69e7f93142ee0d2b98e4/static-site/app/styles/globals.css#L78-L80)

```suggestion
.baseLink:hover {
  color: #31586c;
  text-decoration: inherit;
}
```

---

I'm gonna replicate the `text-decoration: none;` from the base `.baseLink` selector instead, as I think that's more explicit about what we want to have happen. 

Thanks!